18 Normalizr/src/schemas/countrySchema.ts

```diff
@@ -0,0 +1,4 @@
+import { schema } from 'normalizr';
+
+export const countrySchema = new schema.Entity('countries');
```

Comment:
Yep, the index would be a solution in case we need to break down. Btw I meant suffix, not prefix.